### PR TITLE
applications: asset_tracker: Initialize cJSON in cloud codec

### PR DESCRIPTION
**Pull request by jtguggedal**
applications: asset_tracker: Initialize cJSON in cloud codec

Fixes issue where cJSON was never initialized when using the
cloud codec alongside a different cloud backend than nRF Cloud.

Signed-off-by: Jan Tore Guggedal <jantore.guggedal@nordicsemi.no>

---
applications: asset_tracker: Enabled cJSON by default 

cJSON is required for the cloud codec to build.

Signed-off-by: Jan Tore Guggedal <jantore.guggedal@nordicsemi.no>

### CODE REVIEW

File: applications/asset_tracker/src/cloud_codec/cloud_codec.c [hunk 1]
```diff
@@ -721,6 +721,8 @@ int cloud_decode_command(char const *input)
 
 int cloud_decode_init(cloud_cmd_cb_t cb)
 {
+	cJSON_Init();
```
Comment (jhn-nordic) [inline applications/asset_tracker/src/cloud_codec/cloud_codec.c]: Note: CONFIG_CJSON_LIB is selected by CONFIG_NRF_CLOUD if CONFIG_NRF_CLOUD is not set then the user must enalbe CONFIG_CJSON_LIB manually in configs

Comment (jtguggedal) [reply applications/asset_tracker/src/cloud_codec/cloud_codec.c]: I'll add `CONFIG_CJSON_LIB` to the project configs in a commit to this PR

**Comment by jtguggedal** [comment]
Added commit enabling cJSON by default. It's required by the cloud codec to build.

**Comment by rlubos** [comment]
@jtguggedal Could you rebase and resolve the conflicts? We could merge then.

**Comment by jtguggedal** [comment]
@rlubos Thanks, I rebased it now.

**Pull request merged**